roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -10,9 +10,9 @@
   import_tasks: kubeadm-migrate-certs.yml
   when: old_apiserver_cert.stat.exists
 
-- name: kubeadm | Check service account key
+- name: kubeadm | Check apiserver key
   stat:
-    path: "{{ kube_cert_dir }}/sa.key"
+    path: "{{ kube_cert_dir }}/apiserver.key"
   register: sa_key_before
```

Comment:
if you change which key we watch, change the variable name too

Reply:
thanks.